snuba/querylog/__init__.py

```diff
@@ -30,6 +30,8 @@ def _record_timer_metrics(
         metrics,
         tags={
             "status": query_metadata.status.value,
+            "new_status": query_metadata.new_status.value,
```

Comment:
Just a personal preference but I like to use the naming conventions status/legacy_status rather than new_status/status. It makes it easier to deprecate later as it's clearer what should be removed

Reply:
That's fair but I am sort of locked into the `status` name because that's what is used in Datadog. If I change the name then the existing SLO in Datadog will have zero data to start with.

Reply:
There is no `query_status` tag in datadog. That would be better than `new_status`

Reply:
I'll change it to `request_status`.